src/utils/appLinking.js

```diff
@@ -0,0 +1,147 @@
+import { Linking } from "react-native";
+import { Alert } from "../components";
+import {
+	team,
+	general,
+	announcements,
+	fundraisingCommittee,
+	motorshpe,
+	projects20_21,
+	shpejr,
+	mentorshpe20_21,
+	computereng,
+	projectleads,
+	internationalStudents,
+	linkedinDrop
+} from "react-native-dotenv";
+/**
+ * @description Object that holds the slack team id and several channel ids.
+ *              The information in here is the most current as of May 28th, 2020.
+ *              To find the team id or channel id login in slack on a computer.
+ *
+ *  slackInfo
```

Comment:
Remove word "slackInfo" and leave description as the only part of the block comment

Reply:
took care of it.